src/feature/MovieBrowser/Movies/List/PopularMovies/Loading/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
![image](https://user-images.githubusercontent.com/123471023/232242818-29c2c1a5-1bd7-4d3a-9442-7dc873531949.png)
Great job! The css will be defined later :)